src/test/download_bigg_models.sh

```diff
@@ -6,11 +6,13 @@
 #  source ./download_bigg_models.sh
 #################################################
 
+_CWD="$PWD"
 TEST_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 # download and extract the bigg models for testing
-mkdir -p resources/bigg
-cd resources/bigg
+cd TEST_DIR
+mkdir -p $TEST_DIR/resources/bigg
+cd $TEST_DIR/resources/bigg
 wget https://github.com/matthiaskoenig/bigg-models-fba/raw/master/models/bigg_models_v1.5.tar.gz
```

Comment:
@matthiaskoenig Not clear to me why we don't download the BiGG models directly from BiGG. Doesn't this detour introduce the problem that we need to keep two places in sync with BiGG Models Database?